src/Security/Token/BearerTokenInterface.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
-namespace Paysera\BearerAuthenticationBundle\Security\Authentication\Token;
+declare(strict_types=1);
+
+namespace Paysera\BearerAuthenticationBundle\Security\Token;
```

Comment:
Namespace has changed, it's a breaking change for apps that check if token implements this interface. Either keep existing classes where they were or document each change in changelog. IMO 1st option is best.